Library/Formula/supervisor.rb

```diff
@@ -0,0 +1,113 @@
+class Supervisor < Formula
+  desc "Process Control System"
+  homepage "http://supervisord.org/"
+  url "https://github.com/Supervisor/supervisor/archive/3.2.1.tar.gz"
+  sha256 "d9c0b17ec42ac6477e7ef0ad2e1f6f1597f855a4d9606c024ac40eb034d6e9ed"
+
+  depends_on :python if MacOS.version <= :snow_leopard
+
+  resource "meld3" do
+    url "https://pypi.python.org/packages/source/m/meld3/meld3-1.0.2.tar.gz"
+    sha256 "f7b754a0fde7a4429b2ebe49409db240b5699385a572501bb0d5627d299f9558"
+  end
+
+  def install
+    inreplace buildpath/"supervisor/skel/sample.conf" do |s|
+      # A `[unix_http_server]` modification
+      s.gsub! %r{^file=/tmp/supervisor\.sock\b},
+              "file=#{var}/run/supervisor.sock"
+
+      # Some `[supervisord]` modifications
+      s.gsub! %r{^logfile=/tmp/supervisord\.log\b},
+              "logfile=#{var}/log/supervisord.log"
+      s.gsub! %r{^pidfile=/tmp/supervisord\.pid\b},
+              "pidfile=#{var}/run/supervisord.pid"
+
+      # A `[supervisorctl]` modification
+      s.gsub! %r{^serverurl=unix:///tmp/supervisor\.sock\b},
+              "serverurl=unix://#{var}/run/supervisor.sock"
+
+      s.gsub! /^;\[include\]$/,
+              "[include]"
+      s.gsub! %r{^;files = relative/directory/\*\.ini$},
+              "files = #{etc}/supervisor.d/*.ini"
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    resource("meld3").stage do
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
+    # Q.v. <https://github.com/Supervisor/supervisor/issues/608>.
```

Comment:
Not sure what `Q.v` means here; can you elaborate a bit in the comment? Also, I'd avoid the `<..>` wrapping. Thanks!